README.md

```diff
@@ -4,7 +4,7 @@
 
 Please see the [Maintainer wanted issue](https://github.com/activerecord-hackery/ransack/issues/1159) if you are interested.
 
-[![Build Status](https://travis-ci.org/activerecord-hackery/ransack.svg)](https://travis-ci.org/activerecord-hackery/ransack)
+[![Build Status](https://github.com/activerecord-hackery/ransack/workflows/test/badge.svg)](https://github.com/activerecord-hackery/ransack/actions)
```

Comment:
I think it's `https://github.com/activerecord-hackery/ransack/workflows/Test/badge.svg`. With lowercase I get "Not found".

Reply:
Either that or change the workflow name to "test".

Reply:
Right. I prefer to use the lowercase "test".